Dockerfile

```diff
@@ -0,0 +1,44 @@
+##### BASE IMAGE #####
+FROM python:3.6-slim-stretch
+
+##### METADATA ##### I(TODO: Refer cwl-wes)
+LABEL base.image="python:3.6-slim-stretch"
+LABEL version="1.1"
```

Comment:
I copied these LABEL directives from our lab's template, but I don't think they are a bit too many. I would remove this version and instead use the `software.version` one, which you can set to `0.1.0` to start with